test/runtime/manifests/ct-test-policy.json

```diff
@@ -0,0 +1,51 @@
+[{
```

Comment:
Where can we add a description about the behavior this policy offers without reading it through all of it? (Not only for this one but for all present in `manifests` directory)

Reply:
Shouldn't the policy language itself be enough?

Reply:
I mean, it would be useful to get the overall idea of what the test is. The name "ct-test-policy" is not pretty descriptive to me. I was just thinking if we had policies that could be used in multiple tests. Let's imagine there was a policy called "allow client to talk with server on port 80" that could be a policy that could be used in multiple tests for example.

Reply:
We could do this by splitting up the policy into separate files with more descriptive names? Right now, at least with this test, it's somewhat of a hodgepodge of policy rules that make it hard to specify succinctly in the filename itself what it does.